### PR TITLE
add: BitDevs links

### PR DESCRIPTION
**Pull request by joaothallis**

### CODE REVIEW

File: content/en/meetup/2024-10-30.md [hunk 1]
```diff
@@ -4,4 +4,24 @@ location = "TechQuartier, Platz der Einheit 2, floor #3, room name: A66"
...
+  - https://www.nobsbitcoin.com/ashigaru-v1-0-0/
+- Bitcoin Core project disclosed three vulnerabilities fixed in v25.0
+  - [Public disclosure of three vulnerabilities affecting Bitcoin Core <v25.0](https://groups.google.com/g/bitcoindev/c/WeSDeV8YOSA/m/5zE3n2lxAAAJ)
+  - https://x.com/0xB10C/status/1844705899455062220
+- Mining: Recent problems with testnet4
+  - https://x.com/0xB10C/status/1849133671980339313
+  - [miner: Reorg Testnet4 minimum difficulty blocks](https://github.com/bitcoin/bitcoin/pull/31117)
+- Mutation-core: A mutation testing tool for Bitcoin Core
+  - https://delvingbitcoin.org/t/mutation-core-a-mutation-testing-tool-for-bitcoin-core/1119
+
```
Comment (veronikakuett) [inline content/en/meetup/2024-10-30.md:26]: Can you add this one: 
"Lightspark unveiled new L2, called Spark"
https://x.com/kphur/status/1850001832787210365

**Comment by 0xB10C** [comment]
Thanks! I think if we don't end up adding many more links, it should be fine to drop the summary/details part.

File: content/en/meetup/2024-10-30.md (hunk 1 above)
Comment (veronikakuett) [reply content/en/meetup/2024-10-30.md:26]: And maybe this, even though it's already a month old.   
"Soft-Fork/Covenant Dependent Layer 2 Review from Peter Todd"

https://petertodd.org/2024/covenant-dependent-layer-2-review

Comment (0xB10C) [reply content/en/meetup/2024-10-30.md:26]: Will add them!

**Comment by 0xB10C** [comment]
Thanks @joaothallis @veronikakuett!

**Pull request merged**